misc/loop/traefik/letsencrypt/loop/traefik/letsencrypt/acme.json

```diff
@@ -0,0 +1,19 @@
ItH+yQJr8PZ4ZF/ri7Lgoq1CTitO2wN4XqyMIcYiBwDJiII2GF5L6FOgN/PeggAhL1EIJc2NQy44WqXdisbJYS50vaDw1gYMYWWvMuxsl0UbPIpRgr1WmAtQAbXt1C1lhvVBhyJU2npJy6WqIVbwlqXT9nOihHmYbBUZNcaeFrRuhm6XgdD+ISY33Kam6kfbQaPS36x7MW36rEfBV4XWWdUuq5uAZkUPgIMSglYXUO2YCpGHPRuaXww8OAWF3WFzhxQRB01nne1GkSmBz6ST1E6ZFrFpKzKeXkzmsExUU2BGJKa69NdmX+SbYAVeMmrlrh1NdND4P8GdoVPToEiQDs0gN4V3cWrot6aQj7HcG6uC7ojdZIsjQlGHVathWSziT3X4cjqUiBQhFjhAqBp04g60T0ThaZ5jdr4RfVhc9TKV0VSKwqSmmuOrlshl8iUkhF49ar68Grn9PE5hWwXMt5IFA3nnFnJJpljTyeZcm3JSJ1e59RKAFyKBuZvJlr8V1o5j26V34XkNs+uCHlPyI3qTn0PfgNeFi5cnhs8XeOLHz2pckVATkCAwEAAQKCAgAZdKKEdSGFtYDnqofKfyMaNtBnTVMSpzP9nnaKXMPVNGjJJe+0E7msx0ITJoq+lm5vRTf7y7E47e3hgNGaZu3OdCWNWUe1X5inSlIWPotoX2ZTumpceUTmVVPaZ4gcPWtm+ZY/gTIK2b0MAl4chScECnn6H4m1veyP3PmM1FwKpsv0wNJ8Um7Libd6mhC6QJ1/GjAtVspfi6D5F8bxDWzMlRjy7mkU0E5otEy1YNMlz2GaSrtFVCMpMjLU1QJqP4i7wzi5lR13lh06EDC4HDDbYgBmBee46tAEUQee0dweYrRoC3cryOf44IYjPr/4fF1Kyd/ZQ4b+IstYIDL3f3FHYDDoAV57CzhomQ7Dz++WPKFTF1MZ+94sKnX91EGkJof/Wa+Ljde0k4DjPgJIqhpTmyS6ezKx8qaJoRDl+X5aBp8fjhglT0xXPKKpMGHhZihqB1z/D5ybjzo+90UTooM3UZ1eB3MijDM4D62419x8jWI/QjUtfTwYhM6wiOaXIogi38bDNSEWidOdh1vT7CDSzjinX8aetVwW9qkrUhPBt0q1wOVSfoa/K4PwFD3KGjqIhcSpUqQ1diLq02SkrCnF5BH3DEA3BNWQcaD3G23gybbyudRKudUTLofWJqYmCFATydXtFWD2Da3XCUXnALsAgpIVqTm70qtqaAdYGPl8ZQKCAQEAyrFy0L2QO6t+DvTeGzpNJvUss1BOM6W8qW621xTLbEMEn2osJZXgJoZ9296EXCTo54pnOGQpQiu6UTYGMI5AIuu23Hdi+vE901+BChbAyRmkw5hSqo3vwI60F7nzND26NiN453j8jRKqQm9GP1Er9SDdqk0sU1huj1mfIb0puGBjs2ibqb/VSOfrmnrWMvuYQeJUiwEOE1qqdBaTqgtS4gd2ed19vGN/JJiMNgIDrTLL0PzIXy/lAOQKW+WjWA5RIt0DVJlNHlTr4BuSllhxBDXO5p9HLzRjNXUjZETodCWKHU30oq0HDqlZnLXUiKlUowMZc/0+Hd7WSSz65ycsfwKCAQEAwNz70mQs8iqJpGFHqaiYhUXgrERZNAzgS+VEFbkrvMlYAhv2NYF0/NcN40zpp3QiKzfxVYOkuH10FyjmIVxcjSOg23aje+N1zV0e49inKdGC5RF35BX1gk+tjMtUIDG+fUyi1NVWq7jxbqV2Cu00VEZFU79x2Q+dZtEPiYfFh5G1wrmUDASuW69ZD73Kd12/4wnN9qy3AKHztBrNlUx0SImpkWIawKm+HUEL+7m3Jwn/kJdB1rWERdQSvKI2UUUYOLbWqobh9BS/09V/3Fs+TEeb8qn6fDHEY1pnJY95hpqM6yABZgFUfCk62bN5YLjpoXPjQwPo9FVyqZC5PTtWRwKCAQAyINCUb01MdI0ohcaoVTLff6Meda3heAvAm3D8oFfu3mQ2SLvCT96vqHNX/SAe6cHwAXqlp5/MjaP2kbXWLFgQStteiugfx6nSrLvtEtN66Kym8vT+2TuCE8jWVVaghOOYoA02seVDvmn9JCfrYNfON+KaS5u6wRQWv8NrRK0cvDnsFOLqiFbnQPQzsE1pLm/pW2AtQIWKo9GkBDFgzIwi+lWCrhR6kuDO3CtQ8A6YtwdRGDM111J7depZ7+EF1gSa6q34LjG0Xqw3MJqAND8/G0+gGaQxOHOQtPebxK74ipAtzynBUal9JsTSzOVTJw2Xgxy4YtkJ6iJ01nr2z+lhAoIBAA3cU+TM+kX1NKsxqprQ2RVK7lizGM4a1svdkEUXZ8NLnR2ZsNorkaLSOddKIci9Ge+NN7TznxT7Pyqi79cZ8jTaiKb9Ica3lDHMLXomie0Bd6zK0pj70wcofsqNmmAt8wjCRzmQk4gH5KIVFiS243njCvx319DHxIItKDa0IUga+rsh5zft5jIEqhvtge2OOUox3DGMPCK5gPBCtN+z/O9xpVQjftS13knv8NKIiFcv6sQTetHZfcYe9bHE3q58jr8UtNzQkNCf5SFuSKAnlpSY2OZZvPzL/tc9GgijBRLdbJ9BT5hrJA1AUVBwC5sd7vyBfaXFVCqrMXBqTEXQS+UCggEAJqTgNgO1vvE1356MqZ5KKVWD1WUy3QTdzXKwd6lcGdRj+83XdA84YhaWlAH58x2bR5YmpzMdra9poNwH/XopSUV8PhlGVguJKYtxHPjbT14YQQX7m7Oe7HrCZlGCQ4m3zVofDOX3+Cg4yFxhUhNkvVZ37MaCqbDukS/i7KzbTQOg6zfjj0BHHEUYqTde1LqR1/YXYig0ePQu9ECYf+vwwCa2jTfWnkmjiRvd34DX498t2O3cVxHUf93LeHxRt2GFD2cHOR9KaFoy13zY3pnV8d7B6gdyCTBAMeMGsnPSQ64JeYMO/V1ErFQnQwPqEXSxJfQ7KN6Hb372efnW+z3Umg==",
```

Comment:
I suggest not hosting this on Git for two reasons. First, it's a private key that could make TLS inefficient in case of a MITM attack. Second, I believe it changes every 3 months. I recommend adding it to the .gitignore file.

cc @kristovatlas